assignments/array-methods.js

```diff
@@ -56,28 +56,54 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function (runner) {
+    fullName.push(runner.first_name + " " + runner.last_name);
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+let allCaps = runners.map(function(elem, index, array) {
```

Comment:
Using `function(elem, index, array)` (rather than more descriptive names) is likely helpful if you intend to come back to this as example code, but it's a good habit to use your own variable names that are more descriptive.

Also, remember that the only required value for map/reduce/filter/forEach is the `elem`, representing the current item being iterated over -- so if you don't plan to use `index` or `array`, there's no need to include them (unless it's just for practice). It will still work fine, but it might be confusing or make your code appear incomplete if you pass values without using them.